hypha/apply/determinations/templates/determinations/includes/determination_button.html

```diff
@@ -1,6 +1,9 @@
 {% load i18n determination_tags workflow_tags %}
 {% if request.user|show_determination_button:submission %}
-    <a target="_blank" href="{% url 'apply:submissions:determinations:form' submission_pk=submission.id %}" class="button button--primary button--full-width {{ class }}">
+    <a
+        href="{% url 'apply:submissions:determinations:form' submission_pk=submission.id %}"
+        class="button button--primary button--full-width {{ class }}"
+    >
```

Comment:
I'm not fond of `target="_blank"` but it makes some sense here (but less now that we have the option to view the submission in place). It has also been in place for years for determinations and reviews so staff is used to it.

If we make a change it should be in both places.

Reply:
I believe, any person if they really want to have two tabs they would know how to do it, and in most cases they might be just reading the determination and in that case they simply want to view it. 

I'll make an update to reviews.

Reply:
Removed from review button